### DISCUSSION TITLE
Backing up everything , and loading it on another computer

### DISCUSSION THREAD

**Discussion by el3ctr0wqw1** [FunnyWolf/Viper#231] [Q&A] [answered]
Let's imagine a scenario where my first machine where "Viper" is installed suddenly corrupted and no longer works, however i can still browse the files on it, now how to move everything i collected using "Viper" and deploy it on another machine (the machines exploited are the most important thing here). Is it possible to just get the database file and load it on the new machine to recover all the previously exploited machines?

**Comment by FunnyWolf** [comment, marked as answer]
of course, you can just copy VIPER dir to the other machine and run `docker compose up -d` start viper,(all viper database file is store in /root/VIPER/db
